core/src/HIP/Kokkos_HIP.hpp

```diff
@@ -46,6 +46,8 @@ class HIP {
   using size_type    = HIPSpace::size_type;
 
   using scratch_memory_space = ScratchMemorySpace<HIP>;
+  using scratch_memory_space_l0 = ScratchMemorySpaceBase<HIP, L0Tag>;
+    using scratch_memory_space_l1 = ScratchMemorySpaceBase<HIP, L1Tag>;
```

Comment:
```suggestion
  using scratch_memory_space_l0 = ScratchMemorySpaceBase<HIP, L0Tag>;
  using scratch_memory_space_l1 = ScratchMemorySpaceBase<HIP, L1Tag>;
```

Reply:
Sure. I'm more concerned about discussing the interface/design at this point than fixing it all up.

Reply:
just a drive-by as I am trying to get a feel for it :-)